src/main/java/org/jabref/logic/citation/repository/ChainBibEntryRelationsRepository.java

```diff
@@ -0,0 +1,66 @@
+package org.jabref.logic.citation.repository;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Objects;
+
+import org.jabref.model.entry.BibEntry;
+
+public class ChainBibEntryRelationsRepository implements BibEntryRelationsRepository {
+
+    private final BibEntryRelationDAO citationsDao;
+    private final BibEntryRelationDAO referencesDao;
+
+    public ChainBibEntryRelationsRepository(Path citationsStore, Path relationsStore) {
+        this.citationsDao = BibEntryRelationDAOChain.of(
+            LRUCacheBibEntryRelationsDAO.CITATIONS, new MVStoreBibEntryRelationDAO(citationsStore, "citations")
```

Comment:
Make constants for `"citations"`. If it's related to strings that I mentioned previously, then you should use single constant across `ChatinBibEntryRelationsRepository` and `CitationsRelationsTab`